### PR TITLE
WIP Adding e2e kubevirt

### PR DESCRIPTION
**Pull request by bardielle**

### CODE REVIEW

File: ci-operator/step-registry/ipi/conf/kubevirt/install/ipi-conf-kubevirt-install-commands.sh [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+case "${CLUSTER_TYPE}" in
+aws) export AWS_SHARED_CREDENTIALS_FILE=${CLUSTER_PROFILE_DIR}/.awscred;;
+azure4) export AZURE_AUTH_LOCATION=${CLUSTER_PROFILE_DIR}/osServicePrincipal.json;;
+gcp) export GOOGLE_CLOUD_KEYFILE_JSON=${CLUSTER_PROFILE_DIR}/gce.json;;
+kubevirt) export KUBEVIRT_KUBECONFIG=${HOME}/.kube/config;;
+vsphere) ;;
+openstack) export OS_CLIENT_CONFIG_FILE=${CLUSTER_PROFILE_DIR}/clouds.yaml ;;
+openstack-vexxhost) export OS_CLIENT_CONFIG_FILE=${CLUSTER_PROFILE_DIR}/clouds.yaml ;;
+*) echo >&2 "Unsupported cluster type '${CLUSTER_TYPE}'"
+esac
```
Comment (chenyosef) [inline ci-operator/step-registry/ipi/conf/kubevirt/install/ipi-conf-kubevirt-install-commands.sh:29]: Why does a file names ipi-conf-**kubevirt**-install-commands.sh should handle cases of other CLUSTER_TYPES ? I would expect to get the Unsupported cluster type for all types except for kubevirt.

Comment (bardielle) [reply ci-operator/step-registry/ipi/conf/kubevirt/install/ipi-conf-kubevirt-install-commands.sh:29]: This is a side branch for testing our usecase
It's only for testing our image - I must duplicate the commands.sh file but use env variable with our image

**Pull request closed without merging**